server/server.go

```diff
@@ -470,6 +493,41 @@ func (s *Server) getRaft() *raftNode {
 	return r.(*raftNode)
 }
 
+// createActivityStream creates the activity stream and connects a local client
+// that will be subscribed to it.
+func (s *Server) createActivityStream() error {
+	if !s.config.ActivityStream.Enabled {
+		return nil
+	}
+
+	// Connect a local client that will be used to publish on the activity
+	// stream
+	listenAddress := s.config.GetListenAddress()
+	addrs := []string{listenAddress.Host + ":" + strconv.Itoa(s.port)}
+	var err error
+	s.activityStreamClient, err = lift.Connect(addrs)
```

Comment:
Is `createActivityStream` going to get called multiple times during the life of a server? If so, we probably only want to create this client once right? Alternatively, could we just ensure `createActivityStream` gets called only up to one time per server?

Reply:
Ah, later down I see we are closing and niling out `activityStreamClient` when leadership is lost. That is probably sufficient.